### DISCUSSION TITLE
文章中通过链接创建的视频，设置居中无效

### DISCUSSION THREAD

**Discussion by lomiles** [halo-dev/halo#5947] [General]
可以看一下，这个人机对战的视频
http://www.tengogo.com/archives/ren-ji-dui-zhan
![image](https://github.com/halo-dev/halo/assets/35911812/bbac964c-4629-423b-b99e-80badfe543a3)

**Comment by guqing** [comment]
ping @LIlGG

**Comment by LIlGG** [comment]
主题 style 中覆盖了 `img, video` 的设置，将其设置为了块结构，因此会导致内容无法居中。建议向主题提交 issue。

**Reply by lomiles** [reply]
谢谢，提交了
